### ISSUE TITLE
increase code hint window size

### ISSUE THREAD

**Issue by ylazy** [fdorg/flashdevelop#1456]
Hi!
I couldn't find any configuration to make the code hint window bigger.

![codehintwindowsize](https://cloud.githubusercontent.com/assets/6010183/23115655/e182c85c-f778-11e6-8832-e66628c2df99.png)

Is that possible? Please help!
Thanks!

**Comment by Neverbirth** [comment]
Yes, the method call tip should also have a max width increased, we use really long names for some stuff at work and both are sometimes annoying.
